lib/hmmlearn/base.py

```diff
@@ -45,6 +45,27 @@ class ConvergenceMonitor(object):
 
     iter : int
         Number of iterations performed while training the model.
+
+    Examples
+    --------
+    Use custom convergence criteria by subclassing ``ConvergenceMonitor``
+    and redefining the ``converged`` method. The resulting subclass can
+    be used by creating an instance and pointing a model's ``monitor_``
+    attribute to it prior to fitting.
+
+    >>> from hmmlearn.base import ConvergenceMonitor
+    >>> from hmmlearn import hmm
+    >>>
+    >>> class ThresholdMonitor(ConvergenceMonitor):
+    >>>     @property
```

Comment:
fails the doctests.  I guess the continuation lines need to use "...", not ">>>"?

Reply:
TIL! I've never actually run doctest on the examples.

Reply:
neither do it, but it's not my codebase...